.travis.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+echo "Building for Linux..."
+GOOS=linux   GOARCH=amd64 go build
+echo "Building for Darwin..."
+GOOS=darwin  GOARCH=amd64 go build
+echo "Building for FreeBSD..."
+GOOS=freebsd GOARCH=amd64 go build
+
+echo "Running tests.."
+uname -a
+go vet
```

Comment:
We can just define `GOOS=darwin` for `go vet`. I think there is no need for `build`.

Reply:
Hmm I don't think we should assume that `go vet` runs the same checks that `go build` does. Or is this documented?

Reply:
I think this is pretty clear - we cannot rely on it:
> Go vet never builds, so any try build-time flags are pointless. The issue is how the go command identifies packages to pass to go tool vet.
https://github.com/golang/go/issues/21711#issuecomment-326197693

Reply:
We should do both, build and vet

Reply:
sounds good